modin/pandas/indexing.py

```diff
@@ -11,8 +11,8 @@
 import ray
 from warnings import warn
 
-from .utils import (_get_nan_block_id, extractor,
-                    _mask_block_partitions, writer, _blocks_to_series)
+from .utils import (_get_nan_block_id, extractor, _mask_block_partitions,
```

Comment:
I think this change and the change in `utils` should go in a separate PR.

@simon-mo You made these changes would you create that PR?

Reply:
I've reverted these changes